### ISSUE TITLE
Wallpaper is not fully sized when used higher scaling [Issue]

### ISSUE THREAD

**Issue by alok-debnath** [end-4/dots-hyprland#420]
- **(For usage) I have read the [wiki](https://end-4.github.io/dots-hyprland-wiki/en/i-i/02usage)**
- **(For troubleshooting) I have read the [wiki](https://end-4.github.io/dots-hyprland-wiki/en/i-i/03troubleshooting)**
- Linux distro: EndeavourOS

---
## The issue

After the recent changes, i have issues with the wallpaper when setting the scale to higher than 1

<details>
<summary>Click to expand</summary>

`monitor=,preferred,auto,1.2`

![image](https://github.com/end-4/dots-hyprland/assets/67475527/98195b06-a173-4f57-b98b-e37e4a5c1c74)

</details>

**Comment by end-4** [comment]
> After the recent changes

well that's vague
can you play with swww a bit and see if you can fix it

**Comment by alok-debnath** [comment]
`swww query` gives me: 
```bash
eDP-1: 960x540, scale: 2, currently displaying: image: /home/alok/Pictures/42.jpg
```
while `hyprctl monitors` gives:
```bash
Monitor eDP-1 (ID 0):
	1920x1080@59.99900 at 0x0
	description: BOE 0x07F6
	make: BOE
	model: 0x07F6
	serial: 
	active workspace: 3 (3)
	special workspace: 0 ()
	reserved: 0 33 0 0
	scale: 1.20
	transform: 0
	focused: yes
	dpmsStatus: 1
	vrr: 0
	activelyTearing: false
	disabled: false
	currentFormat: XRGB8888
	availableModes: 1920x1080@60.00Hz 1680x1050@60.00Hz 1280x1024@60.00Hz 1440x900@60.00Hz 1280x800@60.00Hz 1280x720@60.00Hz 1024x768@60.00Hz 800x600@60.00Hz 640x480@60.00Hz
```

Does this help, those 2 screen resolution certainly mismatch. The one provided by the `hyprctl monitors` is the correct one

**Comment by alok-debnath** [comment]
> I think this is a bug in swww and was fixed in 0.9.4

Yes thanks, after updating swww to the recent build fixed the issue
